admin/class-primary-term-admin.php

```diff
@@ -105,7 +105,7 @@ protected function get_primary_term( $taxonomy_name ) {
 	protected function get_primary_term_taxonomies( $post_ID = null ) {
 
 		if ( null === $post_ID ) {
-			$post_ID = get_the_ID();
+			$post_ID = $_GET['post'];
```

Comment:
If `get_the_ID()` fails then it's likely that one will fail too, both rely on WP context.